lib/orderly.rb

```diff
@@ -17,8 +17,10 @@ module Orderly
     end
 
     def html_for_node(node)
-      if node.is_a?(Capybara::Node::Document)
-        page.body
+      if node.is_a?(String)
+        node
+      elsif node.is_a?(Capybara::Node::Document)
+        page.find("body").native.inner_html
```

Comment:
This change is breaking the tests in headless chrome. You can just revert this line to `page.body`, this change is not necessary for this to work.

Reply:
The reason for this line is that `page.body` includes some formatting changes. For example, one version (I forget which) does `<img ... />` and the other does `<img ... >` (no `/` on the end).

Reply:
Ah, I understand. Could you try `html_for_node(node.find(:xpath, '//body'))`?

Reply:
Just pushed a commit with that. Let's see if it works.

Reply:
Pushed a commit. Let's see if it works